x-pack/plugins/fleet/server/services/epm/data_streams/get.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchClient } from '@kbn/core/server';
+
+import type { PackageDataStreamTypes } from '../../../../common/types';
+
+import { dataStreamService } from '../../data_streams';
+
+export async function getDataStreams(options: {
+  esClient: ElasticsearchClient;
+  type?: PackageDataStreamTypes;
+  datasetQuery?: string;
+  sortDirection: 'asc' | 'desc';
```

Comment:
nit: we have used `sortOrder` in other places: https://github.com/elastic/kibana/blob/069550d72a5549b0fe45f3290c0d856ea2bfd309/x-pack/plugins/fleet/server/services/agents/crud.ts#L210